### DISCUSSION TITLE
Page Size

### DISCUSSION THREAD

**Discussion by SummerBittencourt** [flet-dev/flet#2391] [Q&A] [answered]
### Question

How can I set the page size?

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by ndonkoHenri** [comment, marked as answer]
[`page.window_height`](https://flet.dev/docs/controls/page#window_height)
[`page.window_width`](https://flet.dev/docs/controls/page#window_width)

**Reply by SummerBittencourt** [reply]
Thanks Buddy

**Comment by jsyoo3722** [comment]
For this kind of questions, I think you should refer to the flet documentation, not issues or discussions here.
They have pretty good search there.
https://flet.dev/docs/controls/page/#height
https://flet.dev/docs/controls/page/#width
https://flet.dev/docs/controls/page/#window_height
https://flet.dev/docs/controls/page/#window_width
